### PR TITLE
Feature/add eventgrid impl

### PR DESCRIPTION
**Pull request by omaramalfi**
# add eventgrid impl

## add eventgrid impl

### CODE REVIEW

File: src/ErniAcademy.Events.ServiceBus/Extensions/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -63,7 +63,7 @@ public static class ServiceCollectionExtensions
             return services;
         }
 
-        internal static IServiceCollection ConfigureOptions<TOptions>(this IServiceCollection services, string sectionKey = SectionKey)
+        internal static IServiceCollection ErniAcademyConfigureOptions<TOptions>(this IServiceCollection services, string sectionKey = SectionKey)
```
Comment (Rabosa616) [inline src/ErniAcademy.Events.ServiceBus/Extensions/ServiceCollectionExtensions.cs]: Why this change? is TOptions a specific type of interface bounded to Erni Academy?

Comment (omaramalfi) [reply src/ErniAcademy.Events.ServiceBus/Extensions/ServiceCollectionExtensions.cs]: done

**Review by Rabosa616 — APPROVED** [review]
